vtStorManaged.ATest/Program.cs

```diff
@@ -29,19 +29,30 @@ static void Main(string[] args)
         {
             uint driveCount;
             eErrorCode errorCode;
+            string path = System.AppDomain.CurrentDomain.BaseDirectory;
+            ILoadRunTimeDll vtStorDll = new cLoadRunTimeDll("vtStor.dll");
+            ILoadRunTimeDll vtStorAtaDll = new cLoadRunTimeDll("vtStorAta.dll");
+            ILoadRunTimeDll vtStorScsiDll = new cLoadRunTimeDll("vtStorScsi.dll");
+            ILoadRunTimeDll vtStorAtaProtocolDll = new cLoadRunTimeDll("vtStorAtaProtocol.dll");
+            ILoadRunTimeDll vtStorScsiProtocolDll = new cLoadRunTimeDll("vtStorScsiProtocol.dll");
+            IRunTimeDll vtStorModule = vtStorDll.Load();
+            IRunTimeDll vtStorAtaModule = vtStorAtaDll.Load();
+            IRunTimeDll vtStorScsiModule = vtStorScsiDll.Load();
+            IRunTimeDll vtStorAtaProtocolModule = vtStorAtaProtocolDll.Load();
+            IRunTimeDll vtStorScsiProtocolModule = vtStorScsiProtocolDll.Load();
 
-            cDriveManagerInterface driveManager = new cDriveManagerInterface();
+            cDriveManagerInterface driveManager = new cDriveManagerInterface(vtStorModule);
```

Comment:
This is where it gets tricky. The user must know which module to go with witch class. We might need to find a better approach.